Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/TextLogFileTest.cs

```diff
@@ -149,9 +147,9 @@ public void TestExclusiveAccess()
 				_file = new TextLogFile(_scheduler, _fname);
 				_scheduler.RunOnce();
 
-				_file.GetValue(LogFileProperties.EmptyReason).Should().Be(ErrorFlags.SourceCannotBeAccessed);
-				_file.GetValue(LogFileProperties.Created).Should().NotBe(DateTime.MinValue);
-				_file.GetValue(LogFileProperties.Created).Should().Be(new FileInfo(_fname).CreationTime);
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.